test/helpers/suite.go

```diff
@@ -47,3 +76,129 @@ var SynchronizedAfterSuiteCallback = func() {
+	}
+	disableDeleteAppStr := os.Getenv("JX_DISABLE_DELETE_APP")
+	disableDeleteApp := "is set. Apps created in the test run will NOT be deleted"
+	if disableDeleteAppStr == "true" || disableDeleteAppStr == "1" || disableDeleteAppStr == "on" {
+		disableDeleteApp = "is not set. If you would like to disable the automatic deletion of apps created by the tests set this variable to TRUE."
+	}
+	disableDeleteRepoStr := os.Getenv("JX_DISABLE_DELETE_REPO")
+	disableDeleteRepo := "is set. Repos created in the test run will NOT be deleted"
+	if disableDeleteRepoStr == "true" || disableDeleteRepoStr == "1" || disableDeleteRepoStr == "on" {
+		disableDeleteRepo = "is not set. If you would like to disable the automatic deletion of repos created by the tests set this variable to TRUE."
+	}
+	disableWaitForFirstReleaseStr := os.Getenv("JX_DISABLE_WAIT_FOR_FIRST_RELEASE")
+	disableWaitForFirstRelease := "is set. Will not wait for build to be promoted to staging"
+	if disableWaitForFirstReleaseStr == "true" || disableWaitForFirstReleaseStr == "1" || disableWaitForFirstReleaseStr == "on" {
+		disableWaitForFirstRelease = "is not set. If you would like to disable waiting for the build to be promoted to staging set this variable to TRUE"
+	}
+	includeAppsStr := os.Getenv("JX_BDD_INCLUDE_APPS")
+	includeApps := "is not set"
+	if includeAppsStr != "" {
+		includeApps = fmt.Sprintf("is set to %s", includeAppsStr)
+	}
+	quiet := os.Getenv("QUIET")
+	if quiet == "" {
+		quiet = "false"
+	} else {
+		quiet = "true"
+	}
+	bddTimeoutBuildCompletes := os.Getenv("BDD_TIMEOUT_BUILD_COMPLETES")
+	if bddTimeoutBuildCompletes == "" {
+		os.Setenv("BDD_TIMEOUT_BUILD_COMPLETES", "20")
+	}
+	bddTimeoutBuildRunningInStaging := os.Getenv("BDD_TIMEOUT_BUILD_RUNNING_IN_STAGING")
+	if bddTimeoutBuildRunningInStaging == "" {
+		os.Setenv("BDD_TIMEOUT_BUILD_RUNNING_IN_STAGING", "10")
+	}
+	bddTimeoutURLReturns := os.Getenv("BDD_TIMEOUT_URL_RETURNS")
+	if bddTimeoutURLReturns == "" {
+		os.Setenv("BDD_TIMEOUT_URL_RETURNS", "5")
+	}
+	bddTimeoutCmdLine := os.Getenv("BDD_TIMEOUT_CMD_LINE")
+	if bddTimeoutCmdLine == "" {
+		os.Setenv("BDD_TIMEOUT_CMD_LINE", "1")
+	}
+	bddTimeoutAppTests := os.Getenv("BDD_TIMEOUT_APP_TESTS")
+	if bddTimeoutAppTests == "" {
+		os.Setenv("BDD_TIMEOUT_APP_TESTS", "60")
+	}
+	bddTimeoutSessionWait := os.Getenv("BDD_TIMEOUT_SESSION_WAIT")
+	if bddTimeoutSessionWait == "" {
+		os.Setenv("BDD_TIMEOUT_SESSION_WAIT", "60")
+	}
+	bddTimeoutDevpod := os.Getenv("BDD_TIMEOUT_DEVPOD")
+	if bddTimeoutDevpod == "" {
+		os.Setenv("BDD_TIMEOUT_DEVPOD", "15")
+	}
+
+	gheUser := os.Getenv("GHE_USER")
+	if gheUser == "" {
+		gheUser = "dev1"
+		os.Setenv("GHE_USER", gheUser)
+	}
+	gheProviderUrl := os.Getenv("GHE_PROVIDER_URL")
+	if gheProviderUrl == "" {
+		gheProviderUrl = "https://github.beescloud.com"
+		os.Setenv("GHE_PROVIDER_URL", gheProviderUrl)
+	}
+
+	utils.LogInfof("jx version:                                         %s\n", version)
```

Comment:
Probably better to always write. What command is that?

Reply:
Actually this might not be so easy. Since Ginko has already started all stdout is already intercepted by it. You would need to run with '-v' in which case things gets really noise. 

I think we talked about this. It feels there is some feature missing here.